### ISSUE TITLE
Tabs with duplicate DisplayName are not allowed

### ISSUE THREAD

**Issue by ndocosta** [microsoftgraph/microsoft-graph-docs#11720]
Currently I'm not able to create a Team with Tabs.
This was working a few days ago, but at least since yesterday it's returning an error, every time I try to specify Tabs on the json.

"message": "Failed to execute Templates backend request CreateTeamFromTemplateRequest. Request Url: https://teams.microsoft.com/fabric/emea/templates/api/team, Request Method: POST, Response Status Code: BadRequest, Response Headers: Strict-Transport-Security: max-age=2592000x-operationid: 6e34c2067695ce49ac9442b72282a77cx-telemetryid: 00-6e34c2067695ce49ac9442b72282a77c-f6a07482381ed246-00X-MSEdge-Ref: Ref A: 89AC0774A15244A382AD25642818581F Ref B: DB3EDGE0921 Ref C: 2021-02-20T14:55:15ZDate: Sat, 20 Feb 2021 14:55:15 GMT, ErrorMessage : {\"errors\":[{\"message\":\"Tabs with duplicate DisplayName are not allowed.\",\"errorCode\":\"Unknown\"},{\"message\":\"'Channel Name' must not be empty.\",\"errorCode\":\"Unknown\"},{\"message\":\"'Channel Name' should not be empty.\",\"errorCode\":\"Unknown\"},{\"message\":\"'Channel Name' must not be empty.\",\"errorCode\":\"Unknown\"},{\"message\":\"'Channel Name' should not be empty.\",\"errorCode\":\"Unknown\"}],\"operationId\":\"6e34c2067695ce49ac9442b72282a77c\"}",

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 14650eac-eef3-e892-50ee-85e4692d21e9
* Version Independent ID: c7aeea07-12b6-45c9-8fb9-f942974811fa
* Content: [Create team - Microsoft Graph v1.0](https://docs.microsoft.com/en-us/graph/api/team-post?view=graph-rest-1.0&tabs=http&source=docs)
* Content Source: [api-reference/v1.0/api/team-post.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/api-reference/v1.0/api/team-post.md)
* Product: **microsoft-teams**
* Technology: **microsoft-graph**
* GitHub Login: @nkramer

**Comment by ariv8** [comment]
Having the same issue, Workflow has been working fine for 1 year and now on friday we're getting those errors. Is Microsoft working on this issue?

**Comment by JarbasHorst** [comment]
Same here 😟 I posted this issue also in Stack Overflow: https://stackoverflow.com/questions/66280832/getting-channel-name-must-not-be-empty-when-creating-microsoft-teams-team-using

**Comment by mdanielcristian** [comment]
Same issue here.

**Comment by ariv8** [comment]
We created a support ticket to Microsoft about this issue, I'll keep you updated when it is resolved

**Comment by AaronKal1** [comment]
This issue is the same for us, we tried creating in both MsGraph explorer as well as Postman using the example provided here https://docs.microsoft.com/en-us/graph/api/team-post?view=graph-rest-1.0&tabs=http 

we received the exact same error message

**Comment by ariv8** [comment]
In the other ticket was resolution to this:
"

This was an issue to do with payload that has channels with tabs - not the name or displayName properties. It has been fixed in MS Graph v1.0.

Also note that both name and displayName on the tabs will work but the documented property is displayName so I suggest you switch to that instead of name.

Here is a sample of a payload I have tested myself. The example in documentation should work with displayName on the tabs configuration.
"

Should be working now
